tests/test_http/LICENSE-2.0.txt

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
```

Comment:
Is this file accidentally included or intentional?

Reply:
Oh, this file is a `local test file`. I use `http://www.apache.org/licenses/LICENSE-2.0.txt` for testing http file system. So I saved a local copy of `LICENSE-2.0.txt` to compare remote and local version of:

`tf.read_file("http://www.apache.org/licenses/LICENSE-2.0.txt")`

`tf.io.read_file("test_http/LICENSE-2.0.txt")`